 - fix some build paths
 - build libpci.a with PIC (yes this sucks, but a non-stable ABI in
    libpic.so sucks too and libpci.a is how Debian goes about it)

--- Makefile
+++ Makefile
@@ -7,10 +7,10 @@
 VERSION=2.2.3
 DATE=2006-05-05
 
-PREFIX=/usr/local
+PREFIX=/usr
 SBINDIR=$(PREFIX)/sbin
 SHAREDIR=$(PREFIX)/share
-IDSDIR=$(SHAREDIR)
+IDSDIR=$(SHAREDIR)/misc
 MANDIR:=$(shell if [ -d $(PREFIX)/share/man ] ; then echo $(PREFIX)/share/man ; else echo $(PREFIX)/man ; fi)
 INSTALL=install
 DIRINSTALL=install -d
--- lib/Makefile
+++ lib/Makefile
@@ -3,6 +3,8 @@
 
 include config.mk
 
+CFLAGS += -fPIC
+
 OBJS=access.o generic.o dump.o names.o filter.o
 INCL=internal.h pci.h config.h header.h sysdep.h types.h
 
